plugin.video.rtk/addon.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.rtk"
+       name="RTK"
+       version="1.0.0"
+       provider-name="RTK">
+  <requires>
+    <import addon="xbmc.python" version="1.0.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="default.py">

```

Comment:
Please set xbmc.python to "2.24.0" for Jarvis. Pls see http://kodi.wiki/view/addon.xml#Dependency_versions for details.